website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -39,6 +39,18 @@ WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE T
 
 ### Upgrading to `0.37.0`+
 
+{{% alert title="Warning" color="warning" %}}
+`0.33.0`+ _only_ supports Karpenter v1beta1 APIs and will not work with existing Provisioner, AWSNodeTemplate or Machine alpha APIs. Do not upgrade to `0.36.0`+ without first [upgrading to `0.32.x`]({{<ref "#upgrading-to-0320" >}}). This version supports both the alpha and beta APIs, allowing you to migrate all of your existing APIs to beta APIs without experiencing downtime.
+{{% /alert %}}
+
+{{% alert title="Warning" color="warning" %}}
+   v0.36.x introduces update to drift that restricts rollback. When rolling back from >=v0.36.0, note that v0.32.9+, v0.33.4+, v0.34.5+, v0.35.4+ are the patch versions that support rollback. If Karpenter is rolled back to an older patch version, Karpenter can potentially drift all the nodes in the cluster.
+{{% /alert %}}
+
+{{% alert title="Warning" color="warning" %}}
```

Comment:
If we have too many warnings, it starts to all be noise. Are there ways to scope down the number of warning messages that we have here?

Reply:
I had a similar thought. Ideally, I think we should probably have a common set of warnings at the top of the page. I think the original concern with this apporach was that users would go right to the version and miss these top level warnings. Maybe if we link to these top level warnings from the version specific section that would be a good compromise?

Reply:
Going to decouple this for the moment, worth consideration though.